src/UNL/UCBCN/Manager/OutputController.php

```diff
@@ -0,0 +1,147 @@
+<?php
+namespace UNL\UCBCN\Manager;
```

Comment:
I wonder how much of this code we could use, possibly by extending the front end controller?